From a7deeac4428f13ef12816da89617fa9e99cfbd2d Mon Sep 17 00:00:00 2001
From: Zidan Wang <zidan.wang@freescale.com>
Date: Wed, 8 Apr 2015 18:00:45 +0800
Subject: [PATCH] MLK-10603 ASoC: imx-sii902x: hdmi audio depends on sii902x
 fb driver

Hdmi audio depends on sii902x fb driver. And add missing back quote.

Signed-off-by: Zidan Wang <zidan.wang@freescale.com>
---
 sound/soc/fsl/Kconfig | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/sound/soc/fsl/Kconfig b/sound/soc/fsl/Kconfig
index e53ffbf..63fb12d 100644
--- a/sound/soc/fsl/Kconfig
+++ b/sound/soc/fsl/Kconfig
@@ -253,8 +253,9 @@ config SND_SOC_IMX_CS42888
 	 a cs42888 codec.
 
 config SND_SOC_IMX_SII902X
-	tristate "SoC Audio support for i.MX boards with sii902x
+	tristate "SoC Audio support for i.MX boards with sii902x"
 	depends on OF && I2C
+	depends on FB_MXS_SII902X
 	select SND_SOC_IMX_PCM_DMA
 	select SND_SOC_FSL_SAI
 	select SND_SOC_FSL_UTILS
-- 
1.8.0

